Frameworks/Foundation/NSCFTimeZone.mm

```diff
@@ -0,0 +1,95 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include "Starboard.h"
+#include "BridgeHelpers.h"
+#include "CFHelpers.h"
+#include "CFFoundationInternal.h"
+#include "NSCFTimeZone.h"
+#include <CoreFoundation/CFTimeZone.h>
+
+#pragma region Factory Class
+@implementation NSTimeZonePrototype
+
+PROTOTYPE_CLASS_REQUIRED_IMPLS(NSTimeZonePrototype)
+
+- (instancetype)initWithName:(NSString*)name {
+    if (!name) {
+        [self release];
+        return nil;
+    }
+
+    return reinterpret_cast<NSTimeZonePrototype*>(
+        static_cast<NSTimeZone*>(CFTimeZoneCreateWithName(kCFAllocatorDefault, static_cast<CFStringRef>(name), false)));
+}
+
+- (instancetype)initWithName:(NSString*)name data:(NSData*)data {
+    if (!name) {
+        [self release];
```

Comment:
For a prototype, this is a no-op. It can stay, but it might be confusing that we're only releasing ourselves in one codepath (since neither codepath returns self)